utils/plot-utils.ipynb

```diff
@@ -29,20 +29,32 @@
     "    .register((figure, ctx) -> {\n",
     "        ctx.renderIfRequested(io.github.spencerpark.jupyter.kernel.display.mime.MIMEType.TEXT_HTML, () -> {\n",
     "            String id = UUID.randomUUID().toString().replace(\"-\", \"\");\n",
-    "            \n",
+    "            boolean plotImage = System.getenv().containsKey(\"D2L_PLOT_IMAGE\");\n",
```

Comment:
we should check D2L_PLOT_IMAGE value here, otherwise `export D2L_PLOT_IMAGE=0` is still treated as `export D2L_PLOT_IMAGE=1`

Reply:
I think it should be fine since we only use that internally for book generation